src/federation/inbox.ts

```diff
@@ -173,6 +174,18 @@ export async function onFollowAccepted(
     inboxLogger.debug("Invalid actor: {actor}", { actor });
     return;
   }
+
+  const isRelayAccept = accept.objectId?.hash.includes("#relay-follows/");
```

Comment:
Wouldn't it be more appropriate to use `startsWith()` instead of `includes()` here?